### DISCUSSION TITLE
Picture element image

### DISCUSSION THREAD

**Discussion by SirUlbrich** [Tom-Bom-badil/samson_trovis_557x#50] [General (de/en)]
Hallo @Tom-Bom-badil , kannst du mir sagen womit du die Bilder erstellt hast? Meine Anlage ist leider nicht so aufgebaut und würde es gerne abändern. 
Dankeschön ☺️

**Comment by Tom-Bom-badil** [comment]
Ursprünglich in Visio mit meheren Layern, jetzt in CorelDraw. Alternativ sollte auch das freie Inkscape gute Dienste leisten.

/tom
